packages/ses/scripts/bundle.js

```diff
@@ -3,20 +3,21 @@ import '../index.js';
 import fs from 'fs';
 import { makeBundle } from '@endo/compartment-mapper/bundle.js';
 import terser from 'terser';
+import { fileURLToPath, pathToFileURL } from 'url';
 
-const resolve = (rel, abs) => new URL(rel, abs).toString();
-const root = resolve('..', import.meta.url);
+const resolve = (rel, abs) => fileURLToPath(new URL(rel, abs));
+const root = new URL('../', import.meta.url);
```

Comment:
Nit: `root` is changing from a `string` to a `URL`. This is fine since the only usage is with `resolve` which passes it to `new URL()`, which accepts either a `string` or `URL`, but since we don't have explicit typing in this file, I usually prefer to also rename identifiers to reflect the type held.

Reply:
Adding liberal toStrings.

Reply:
If converting to `toString` is there any reason not to keep the original `resolve('..', import.meta.url)` ? I assume that URL does the right thing and has a trailing `/` after `..` resolution ?

Reply:
`fileToPath` seems to be sensitive to the final slash indication.

Reply:
Checked, it’s not sensitive to the final slash.